lib/ruby_parser_extras.rb

```diff
@@ -723,7 +723,9 @@ def new_body val
       result = block_append(result, elsebody)
     end
 
-    result = s(:ensure, result, ensurebody).compact.line result.line if ensurebody
+    if ensurebody
+      result = s(:ensure, result, ensurebody).compact.line (result || ensurebody).line
```

Comment:
I split this out to a local variable because I found the "space open-paren" part confusing.

Reply:
Absolutely :+1: